librz/core/linux_heap_glibc.c

```diff
@@ -188,7 +188,14 @@ static void GH(update_arena_without_tc)(GH(RzHeap_MallocState) * cmain_arena, Ma
 	main_arena->GH(max_system_mem) = cmain_arena->max_system_mem;
 }
 
-static bool GH(update_main_arena)(RzCore *core, GHT m_arena, MallocState *main_arena) {
+/**
+ * @brief Store the MallocState struct of an arena with base address m_arena in main_arena
```

Comment:
In other places we use `\brief` syntax instead of `@brief`. Please use it consistently. See https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md#documentation